modules/autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/viewing/AutofillManagementEditMode.kt

```diff
@@ -142,3 +140,8 @@ class AutofillManagementEditMode : Fragment() {
             }
     }
 }
+
+private fun String.convertBlankToNull(): String? {
+    if (this.isBlank()) return null
```

Comment:
NIt: could b e simplified to:
`
private fun String.convertBlankToNull(): String? = if (this.isBlank()) null else this
`